docs/lua-api-dimension.md

```diff
@@ -35,27 +102,93 @@ gravity = 0.5
 
 Default value is `1`.
 
-### `id`
+#### `horizontal_sprint_mod`
 
-ID of the dimension. **Mandatory field.**
+Horizontal sprint speed multiplier.
 
 Example:
 ```lua
-id = "dimension_nether"
+horizontal_sprint_mod = 1.5
 ```
 
-IDs are usually of the form `mod:dimension` but the `mod:` prefix is prepended automatically so it's not needed.
+Default value is `1.5`.
 
-### `name`
+#### `is_sneak_always_mod`
 
-Label of the dimension. **Mandatory field.**
+Whether sneaking applies the horizontal mod while airborne.
 
 Example:
 ```lua
-name = "Nether"
+is_sneak_always_mod = false
 ```
 
-This label is the name that will appear everywhere in the game.
+Default value is `false`.
+
+#### `jump_antigravity`
+
+Gravity is decreased by this amount while holding the jump key prior to the peak of the jump.
+
+Example:
+```lua
+jump_antigravity = 0.04
+```
+
+Default value is `0.04`.
+
+#### `jump_speed`
+
+Jump speed of the dimension.
+
+Example:
+```lua
+jump_speed = 0.3
+```
+
+Default value is `0.3`.
+
+#### `move_speed`
+
+Movement speed in this dimension.
+
+Example:
+```lua
+move_speed = 0.04
+```
+
+Default value is `0.04`.
+
+#### `sneak_horizontal_mod`
+
+Horizontal movement speed is multiplied by this amount while sneaking.
+
+Example:
+```lua
+sneak_horizontal_mod = 0.1
+```
+
+Default value is `0.1`.
+
+#### `sneak_vertical_speed`
```

Comment:
Shouldn't this be `sneak_verical_mod` to match the style of the other modifiers?

Reply:
Ah, it seems I got lazy and didn't realize what it did when writing the explanation. Sneak Vertical Speed applies downwards momentum while Sneaking, which is a migrated feature from vanilla